fabric/v4/model_route_table_entry_filters.go

```diff
@@ -3,7 +3,7 @@
  *
  * Equinix Fabric is an advanced software-defined interconnection solution that enables you to directly, securely and dynamically connect to distributed infrastructure and digital ecosystems on platform Equinix via a single port, Customers can use Fabric to connect to: </br> 1. Cloud Service Providers - Clouds, network and other service providers.  </br> 2. Enterprises - Other Equinix customers, vendors and partners.  </br> 3. Myself - Another customer instance deployed at Equinix. </br>
  *
- * API version: 4.9
+ * API version: 4.10
```

Comment:
Out of scope for this PR, but it might be nice to leave this version out of the header comment in the generated code files (or to remove the header comment entirely, since it seems at a glance to be the same across all files).  That would eliminate some unnecessary changes in future sync PRs and make it easier to focus on the actual code changes.

Reply:
We've made a comment and will add that to the backlog for the next release. Great comment 👍